_assets/javascripts/components/bitmovin.js

```diff
@@ -0,0 +1,248 @@
+/* global CRDS */
+/* global moment */
+class BitmovinManager {
+  constructor(bitmovinConfig) {
+    this.isCard = bitmovinConfig.isCard;
+    this.isStream = bitmovinConfig.isStream;
+    this.videoDuration = Number(bitmovinConfig.duration) * 1000 ;
+    this.timezoneStr = 'America/New_York';
+    this.container = document.getElementById(`${bitmovinConfig.id}`);
+    this.countdown = new CRDS.Countdown();
+    this.playerConfig = {
+      key: '224f523d-e1ba-4f96-ad4d-96365f461c93',
+      playback: {
+        autoplay: this.getAutoPlay(),
+        muted: this.getIsMuted()
+      },
+      analytics: {
+        key: '01e90136-7623-4df4-b0d9-a3d975b00258',
```

Comment:
same here?

Reply:
After thinking about it, we might want to let bitmovin know we exposed these (PR's are forever) and see if that has any implications for driving up our bill if someone were to misuse it.